### ISSUE TITLE
Miss-chance based on FOV

### ISSUE THREAD

**Issue by SylveonBottle** [nullworks/cathook#727]
Would be pretty cool, would probably look legit as well.

If the user's aim FOV is set to 10, then a target's head at the 9(fov) mark would have a 10% of being shot by aimbot.
An enemy within 2(fov) would have 80% headshot rate. 

Would force user to at least try and be more accurate, and would result in less callouts due to first person spectating.

Idk, just random thought I had.

**Comment by EricSmith17** [comment]
you mean target selection by distance to crosshair ?

**Comment by TotallyNotElite** [comment]
> 
> 
> you mean target selection by distance to crosshair ?

No. The further your crosshair is from the hitbox, the higher the chance of a miss should be when using miss chance.
